HACKING.md

```diff
@@ -0,0 +1,29 @@
+
+#Hacking On The Manifesto
+
+###Background
+
+The manifesto acts as a sort of code of conduct for developers down with the Namespace App Club movement. While we won't
```

Comment:
App club doesn't really clarify things because a Google search doesn't surface anything relevant and I'm not familiar with the term being used elsewhere. If we would like to introduce Namespace to people who would be encountering us for the first time we can briefly describe in one sentence what Namespace is. A description would work much better than a name.